chapter-03/recipe-10/c-example/hwclient.c

```diff
@@ -1,27 +0,0 @@
-//  Hello World client
```

Comment:
This is the correct version of `hwclient.c` since it will work on UNIX and Windows.